### DISCUSSION TITLE
hot keys for track view

### DISCUSSION THREAD

**Discussion by Reaper10** [tenacityteam/tenacity-legacy#724] [Ideas]
I think that you should a hot keys for track view options.

<img width="156" alt="Capture" src="https://user-images.githubusercontent.com/8137561/172475908-84e2f22a-87ff-4bfb-907a-771ff0d3e3d2.PNG">
 like ctrl 1- 3

**Comment by n0toose** [comment]
Dear @Reaper10,

I seriously wholeheartedly appreciate your enthusiasm and your persistence with reporting bugs to us, but at this point, it may be reasonable to bring up that the development of our audio editor has been stalled for a few months now and is currently on hiatus, as we also openly mention in our organization's description and in this repository's description.

Your ideas are great, but I just want to let you know that spending your free time and effort on contributing to the project like that may not be worth it as your contributions will only effectively matter if there is a development team that will deliberate them, but we do not currently have any longer-term plans for development as a result of burnout, a very weak and exhausting start that ended up biting us multiple times later down the line, lots of abusive behavior from different community members and us moving on to other projects.

I am clarifying all of this just in case many people end up reading the response that I am making.

Best regards,
@n0toose
